core/dbt/tests/fixtures/project.py

```diff
@@ -8,12 +8,51 @@
 
 import dbt.flags as flags
 from dbt.config.runtime import RuntimeConfig
-from dbt.adapters.factory import get_adapter, register_adapter, reset_adapters
+from dbt.adapters.factory import get_adapter, register_adapter, reset_adapters, get_adapter_by_type
 from dbt.events.functions import setup_event_logger
-from dbt.tests.util import write_file, run_sql_with_adapter, TestProcessingException
+from dbt.tests.util import (
+    write_file,
+    run_sql_with_adapter,
+    TestProcessingException,
+    get_connection,
+)
 
 
 # These are the fixtures that are used in dbt core functional tests
+# The main functional test fixture is the 'project' fixture, which combines
+# other fixtures to write out a dbt_project in a temporary directory.
```

Comment:
Should we call out that the fact that the dbt_project in a temporary dir is a by-product/effect of the fixture but not the fixture itself?